Hadrons/Global.hpp

```diff
@@ -272,7 +272,7 @@ struct Correlator: Serializable
 {
     GRID_SERIALIZABLE_CLASS_MEMBERS(ARG(Correlator<Metadata, Scalar>),
                                     Metadata,             info,
-                                    std::vector<Complex>, corr);
+                                    std::vector<Scalar>, corr);
```

Comment:
Not sure why this shows up as a change again... Should have been merged already?